core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -931,6 +931,17 @@
    */
   public static final String LOCAL_DICTIONARY_ENABLE_DEFAULT = "false";
 
+  /**
+   * System property to enable or disable local dictionary generation
+   */
+  public static final String LOCAL_DICTIONARY_SYSTEM_ENABLE = "carbon.local.dictionary.enable";
+
+  /**
+   * System property to enable or disable local dictionary generation default value
+   */
+  public static final String LOCAL_DICTIONARY_SYSTEM_ENABLE_DEFAULT = "false";
```

Comment:
LOCAL_DICTIONARY_ENABLE_DEFAULT itself is enough for default value

Reply:
ok, removed